### DISCUSSION TITLE
Give the configurable option to make RandomTest truly random

### DISCUSSION THREAD

**Discussion by Greenknight9000** [ikemen-engine/Ikemen-GO#938] [Feature Request] [answered]
**Is your feature request related to a problem? Please describe.**
I like to have RandomTest mode on as a "Screensaver" of sorts - just keep something active on my screen while I'm busy, however I noticed that I'd constantly be getting the same matchups again and again. I eventually realized that if a fighter wins the battle, they'll be retained for the next match (E.G.: Sonic Vs Goku - Sonic wins the matches and a new fight starts, the opponent is different, but Sonic will still be one of the fighters) and as I have quite a few powerful characters (and even a few Overpowered Characters, such as Hyper Sonic) the chances are that the powerful characters will be the ones appearing time and time again.

**Describe the solution you'd like**
An option in the settings menu or System.def that changes match loading in RandomTest, such as the following:
RandomTest.WinnerStays = 1/0
RandomTest.LoserStays = 1/0
RandomTest.SwitchOnDraw = 1/0

**Comment by K4thos** [comment, marked as answer]
no plans for that. RandomTest is not meant to be random matches but testing AIs. Mode like this can be implemented via external modules.
